examples/image_streamer_artifact_bundle.yml

```diff
@@ -0,0 +1,117 @@
+###
+# Copyright (2016) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+---
+- hosts: all
+  vars:
+    config: "{{ playbook_dir }}/oneview_config.json"
+    artifact_bundle_name: 'Artifact Bundle'
+    artifact_bundle_name_update: 'Artifact Bundle Updated'
+    artifact_bundle_description: 'Description of Artifact Bundles Test'
+    artifact_download_path: '~/downloaded_artifact.zip'
+    artifact_archive_download_path: '~/downloaded_archive.zip'
+    deployment_groups_uri: '/rest/deployment-groups/c5a727ef-71e9-4154-a512-6655b168c2e3'
+    local_artifact_path: '~/uploaded_artifact.zip'
+    local_backup_artifact_bundle_path: '~/uploaded_backup.zip'
+    build_plans_uri: '/rest/build-plans/ab65bb06-4387-48a0-9a5d-0b0da2888508'
+    build_plans_read_only: 'false'
```

Comment:
Why have you extracted that to a var?

Usually we define on the playbook vars which the users should change the values to be able to run the playbook in their own environment, for example an URI of existing resources, a filepath to upload, an IP address, etc.

I believe that those vars don't need to be changed according to the user's environment, and could be hardcoded in the ansible plays:

* artifact_bundle_name
* artifact_bundle_name_update
* artifact_bundle_description
* build_plans_read_only.